tests/dashbio_demos/app_manhattan_plot.py

```diff
@@ -18,7 +18,13 @@
 DATAPATH = os.path.join(".", "tests", "dashbio_demos", "sample_data", "manhattan_")
 
 # Load the data
-DATASET = pd.read_csv("{}data.csv".format(DATAPATH))
+DT = pd.read_csv("{}data.csv".format(DATAPATH))
+
+# Trim down the data
+datasets = [DT.loc[DT['CHR'] == i+1] for i in range(23)]
```

Comment:
Out of curiosity, wondering if `DT.where(DT.CHR == i + 1)` would be equivalent...

Reply:
I don't think so -- `DataFrame.where` *replaces* the value in the dataframe with `NaN` (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.where.html). Here, I'm trying to split the data by chromosome, then trimming down the data for each chromosome separately, then putting them back together to result in a smaller dataset.

Reply:
Oh, right, it's a mask. Thanks for double checking!